third.go

```diff
@@ -0,0 +1,65 @@
+package main
+
+import (
+	"fmt"
+	"sort"
+)
+
+func Average(inputArray []int) float64 {
+	return float64(CalculateSumOfArray(inputArray)) / float64(len(inputArray))
+}
+
+func CalculateSumOfArray(inputArray []int) (result int) {
```

Comment:
Minor: we are dealing with slice here, but variable name and func name contain `array` word.